.github/workflows/pr_commands.yaml

```diff
@@ -0,0 +1,100 @@
+name: PR Commands
```

Comment:
Do we need to also add this action to other repos (e.g. `kubeflow/trainer`) @thesuperzapper ?